docs/config/1.1/config.jsonc

```
@@ -0,0 +1,73 @@
+// Sundry config.json 版本 1.1 示例
+// 相关配置项的 key 必须使用小写
+
+{
+    "$schema": "https://duckduckstudio.github.io/yazicbs.github.io/Tools/Sundry/config/schema/1.1.json",
+
+    // 配置文件的版本。
+    // old: sundry-old 的配置文件
+    // develop: DuckDuckStudio/Sundry main 分支最新提交的版本
+    // 1.0: Sundry 1.4.1 及以前的版本
+    // 1.1: Sundry 1.4.2 及后续的版本
+    "version": 1.1, // str
```

```suggestion
    "version": "1.1", // str
```

它应该是一个 string 而不是 number。